src/main.jsx

```diff
@@ -70,19 +66,15 @@ function mixItemProps(style, props) {
     divStyle.flexGrow = 0;
     divStyle.flexShrink = 0;
   }
-  
-  if (props.style) {
-    return Object.assign({}, itemStyle, divStyle, props.style);
-  } else {
-    return Object.assign({}, itemStyle, divStyle);
-  }
+
+  return Object.assign({}, itemStyle, divStyle, style);
```

Comment:
1. `style` argument was unused and now it is used
2. Object.assign is okay if you pass `undefined` in it. No need to check